bin/clover/src/pipeline-steps/prettifySocketNames.ts

```diff
@@ -40,6 +40,6 @@ function toSpaceCase(name: string) {
   return name
     // separate any sequence of lowercase letters followed by an uppercase letter
     .replace(/([a-z])([A-Z])/g, "$1 $2")
-    // Separate any sequence of more than 1 of uppercase letters (acronyms) from the next word
-    .replace(/([A-Z]+)([A-Z][a-z])/g, "$1 $2");
+    // Separate any sequence of more than 3 of uppercase letters (acronyms) from the next word
+    .replace(/([A-Z]{3,})([A-Z][a-z])/g, "$1 $2");
```

Comment:
I assume this is to fix some props that had ID in uppercase? Or are there other things too? Not sure how to think about what this will impact. Could probably use that context in the comment.

Reply:
This was causing us to set things like `VPCs` to `VP Cs`. I'm favoring the three letter acronym as AWS tends to use them

Reply:
I see, so your pattern fixes that case but still separates things like VPCId -> VPC Id.